sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -203,14 +203,14 @@ int gnrc_netif_get_from_netdev(gnrc_netif_t *netif, gnrc_netapi_opt_t *opt)
             break;
 #endif  /* GNRC_IPV6_NIB_CONF_ROUTER */
 #endif  /* MODULE_GNRC_IPV6 */
-#ifdef MODULE_GNRC_SIXLOWPAN_IPHC
-        case NETOPT_6LO_IPHC:
+#ifdef MODULE_GNRC_LOWPAN_IPHC
```

Comment:
This module doesn't exist (and I would prefer to keep the name `gnrc_sixlowpan_iphc` or rename it to `gnrc_sixlo_iphc`).